tripal_chado/src/Plugin/TripalImporter/GFF3Importer.php

```diff
@@ -394,22 +395,26 @@ public function form($form, &$form_state) {
     // Always call the parent form to ensure Chado is handled properly.
     $form = parent::form($form, $form_state);
 
-    // get the list of organisms
-    $organisms = chado_get_organism_select_options(FALSE, TRUE);
+    $settings = \Drupal::config('tripal.settings');
 
     // get the sequence ontology CV id
     $conditions = ['cv.name' => 'sequence'];
     $cv_records = $this->cvterm_buddy->getCv($conditions, []);
     $sequence_cv_id = $cv_records[0]->getValue('cv.cv_id');
 
-    $form['organism_id'] = [
-      '#title' => t('Existing Organism'),
-      '#type' => 'select',
-      '#description' => t("Choose an existing organism to which the entries in the GFF file will be associated."),
-      '#required' => TRUE,
-      '#options' => $organisms,
-      '#empty_option' => t('- Select -'),
-    ];
+    $options = [];
+    // Set some defaults to keep each of the fields simpler.
+    $options['select_limit'] = $settings->get('tripal_entity_type.widget_global_select_limit') ?? 50;
+    $options['match_limit'] = $settings->get('tripal_entity_type.match_limit') ?? 10;
+    $options['match_operator'] = $settings->get('tripal_entity_type.match_operator') ?? 'CONTAINS';
+    $options['size'] = $settings->get('tripal_entity_type.size');
+    $options['placeholder'] = $settings->get('tripal_entity_type.placeholder');
```

Comment:
Same comment as in the other importer. If these are moved into the form elements in the controller as defaults then they can be removed here.